app/src/main/java/com/arygm/quickfix/ui/home/MessagesScreen.kt

```diff
@@ -176,9 +426,9 @@ fun Header(navigationActions: NavigationActions) {
 }
 
 @Composable
-fun MessageBubble(message: Message, isSent: Boolean) {
+fun MessageBubble(message: Message, isSent: Boolean, modifier: Modifier = Modifier) {
   Row(
-      modifier = Modifier.fillMaxWidth().padding(vertical = 10.dp, horizontal = 8.dp),
+      modifier = modifier.fillMaxWidth().padding(vertical = 10.dp, horizontal = 8.dp),
```

Comment:
Small absolute values here, you can pass `maxWidth` and `maxHeight` as parameters of this composable and use widthRatio and HeightRatio to make padding responsive :)

Reply:
You're right, but since these are just small values, I think it's fine as is.